sensu/plugins/check-vyatta-cpu.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+#
+# Checks CPU usage
+#
+# Return WARNING if CPU usage is over 80%
+# Return CRITICAL if CPU usage is over 95%
```

Comment:
These feel like things we may want to be able to adjust with inputs rather than having it hard coded in the check.